dffml/cli/dataflow.py

```diff
@@ -92,9 +96,29 @@ async def run(self):
                 print((await loader.dumpb(exported)).decode())
 
 
+@config
+class RunCMDConfig(SourcesCMDConfig):
+    # Just to get the sources working for now. Will probably change it later
+    sources: Sources = field(
+        "Sources for loading and saving",
+        default_factory=lambda: Sources(
+            JSONSource(
+                FileSourceConfig(
+                    filename=os.path.join(
+                        os.path.expanduser("~"), ".cache", "dffml.json"
+                    )
+                )
+            )
+        ),
+        metadata= {
```

Comment:
Since these are going to be very common, let's have field put them in metadata for us to shorten the amount of code we're writing

```python

def field(description: str, *args, required: bool = False, ... (position and labeled here too) ... , metadata: Optional[dict] = None, **kwargs):
    """
    Creates an instance of :py:func:`dataclasses.field`. The first argument,
    ``description`` is the description of the field, and will be set as the
    ``"description"`` key in the metadata ``dict``.
    """
    if not metadata:
        metadata = {}
    #  Add code for positional arguments
    metadata["description"] = description
    metadata["required"] = required
    return dataclasses.field(*args, metadata=metadata, **kwargs)
```

Reply:
Okay!